src/containers/shared/components/Transaction/EscrowCancel/Simple.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+import { Amount } from '../../Amount'
```

Comment:
You should delete `src/containers/Transactions/test/mock_data/EscrowCancel.json`